Vic2ToHoI4/Source/HOI4World/Decisions/Decision.cpp

```diff
@@ -22,6 +22,10 @@ HoI4::decision::decision(std::string decisionName, std::istream& theStream):
 		const commonItems::stringOfObject theHighlightStates(theStream);
 		highlightStates = theHighlightStates.getString();
 	});
+	registerKeyword("highlight_state_targets", [this](const std::string& unused, std::istream& theStream) {
```

Comment:
This is prone to bugs. Rather use a regex with | separated keywords.

Reply:
Now redundant, as the old is removed.